dockers/docker-base/Dockerfile.j2

```diff
@@ -25,12 +25,14 @@ RUN apt-get clean && apt-get update
 RUN apt-get -y install \
     rsyslog            \
     supervisor         \
-    vim-tiny           \
+    vim                \
```

Comment:
given the smallest disk we have to deal with is still 1.5GB, i recommend to keep vim in the debug build.